src/CashuWallet.ts

```diff
@@ -214,15 +216,18 @@ class CashuWallet {
 		amount: number,
 		proofs: Array<Proof>,
 		options?: {
-			preference?: Array<AmountPreference>;
+			preference?: Preferences;
```

Comment:
How about we deprecate `preference`, do the new thing in a new option called `outputAmounts`, and write a thin translation method that maps `preference` to `outputAmounts`, and issue a deprecation warning?

Reply:
I second this